lib/std/json.zig

```diff
@@ -1573,11 +1573,16 @@ fn parseInternal(comptime T: type, token: Token, tokens: *TokenStream, options:
                                     //     .UseLast => {},
                                     // }
                                     if (options.duplicate_field_behavior == .UseFirst) {
+                                        // unconditonally ignore value. for comptime fields, this skips check against default_value
+                                        parseFree(field.field_type, try parse(field.field_type, tokens, options), options);
+                                        found = true;
```

Comment:
found would already be `true` to get here

Reply:
`fields_seen[i]` is true but `found` is only set to true immediately before breaking out of the inline for loop over the struct fields.